src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -163,7 +164,15 @@ method getChatColor*(self: Module): string =
   return chatDto.color
 
 method amIChatAdmin*(self: Module): bool =
-  return false
+  if(not self.controller.belongsToCommunity()):
+    let chatDto = self.controller.getChatDetails()
+    for m in chatDto.members:
+      if (m.id == singletonInstance.userProfile.getPubKey()):
```

Comment:
I guess this is for Chat Groups. Chat groups have a single admin. There should be a property for that. Being a member of a group does not make you an admin.